src/app/modules/vault-filter/components/organization-filter.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component } from "@angular/core";
+
+import { OrganizationFilterComponent as BaseOrganizationFilterComponent } from "jslib-angular/modules/vault-filter/components/organization-filter.component";
+import { DisplayMode } from "jslib-angular/modules/vault-filter/models/display-mode";
+
+@Component({
+  selector: "app-organization-filter",
+  templateUrl: "organization-filter.component.html",
+})
+export class OrganizationFilterComponent extends BaseOrganizationFilterComponent {
+  get show() {
+    const hiddenDisplayModes: DisplayMode[] = ["singleOrganizationAndPersonalOwnershipPolicies"];
+    return (
+      !this.hide &&
+      this.organizations.length > 0 &&
+      hiddenDisplayModes.indexOf(this.displayMode) === -1
```

Comment:
Check my understanding here: this hides the org filter component if:
* there are no organizations - there's nothing to filter by, it's all your vault
* the single org + personal ownership policies apply - there's nothing to filter by, it's all the single org's vault

Is that right?

Reply:
Correct. This design changed over the course of development, which is why you caught some relics like the templates for `noOrganizations` and the policy combo existing that have now been removed.